pkg/agent/route/route_linux.go

```diff
@@ -633,7 +633,8 @@ func (c *Client) AddRoutes(podCIDR *net.IPNet, nodeName string, nodeIP, nodeGwIP
 		// Set the peerNodeIP as next hop.
 		route.Gw = nodeIP
 	} else {
-		// NoEncap traffic to Node on the same subnet. It is handled by host default route.
+		// NetworkPolicyOnly mode or NoEncap traffic to a Node in a different subnet.
```

Comment:
I think "on a different subnet" is correct, we also use it in the comment above

Reply:
I feel "in subnet" is used much more than "on subnet" (for example, that matches the google search results within site:aws.amazon.com). But I do not mind change back to "on" if you prefer.